unstructured/file_utils/filetype.py

```diff
@@ -489,13 +499,16 @@ def wrapper(*args, **kwargs):
             include_metadata = params.get("include_metadata", True)
             if include_metadata:
                 metadata_kwargs = {
-                    kwarg: params.get(kwarg) for kwarg in ("include_page_breaks", "filename", "url")
+                    kwarg: params.get(kwarg) for kwarg in ("filename", "url", "text_as_html")
```

Comment:
Do we not need `include_page_breaks` in here any longer? Just double checking that was intentional

Reply:
Since `_add_element_metadata` doesn't handle page breaks anymore and `include_page_breaks` isn't actually stored in the metadata, we don't need to pass it.